modules/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"devDependencies": {
-		"@liferay/npm-scripts": "37.0.0",
+		"@liferay/npm-scripts": "37.1.2",
```

Comment:
So 37.1.2 is all good as-is, right?

We just need to follow up with the preset-moving stuff to make "some higher version work" for the old branches again, right?

Reply:
Yes. `37.1.2` only works with this PR, but is correct. Then, we will refactor it with the presets stuff and retest when we merge it in master.

Reply:
After that, we can backport (hopefully :candle: ).